internal/docker/client.go

```diff
@@ -0,0 +1,230 @@
+package docker
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+	mysqld "github.com/go-sql-driver/mysql"
+)
+
+const defaultPass = "pass"
```

Comment:
Can't we generate it on on creation?

Reply:
I don't see the need to. It is just temprorary. Password does not really matter. It is needed for the various databases.